extension/src/test/Common.test.ts

```diff
@@ -16,4 +16,17 @@ suite("Common", function () {
       "Incorrect date string returned"
     );
   });
+  test.only("orderedJsonStringify", function () {
```

Comment:
heheh

Reply:
there should be a way to block commit of some code... like `test.only`. 😣